### PR TITLE
added duck api cmd

### PR DESCRIPTION
**Pull request by sougataghar477**

### CODE REVIEW

File: commands/duck.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const got = require('got');
+
+module.exports = {
+	name: 'dog',
```
Comment (JoachimFlottorp) [inline commands/duck.js:4]: duck

File: commands/duck.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	permission: 100,
+	category: 'Random command',
+	execute: async (channel, user, input, perm) => {
+		try {
+			if (module.exports.permission > perm) {
+				return;
+			}
+			const image = await got('https://random-d.uk/api/v2/random', { timeout: 10000 }).json();
+
+			return `DuckerZ quack ${image.message}`;
```
Comment (JoachimFlottorp) [inline commands/duck.js:16]: message should be changed to url

File: commands/duck.js (hunk 1 above)
Comment (hotbear1110) [reply commands/duck.js]: Fixed in 4ecaa183ede7d7ffc730ff4b5c1b08666f0d613d

File: commands/duck.js (hunk 2 above)
Comment (hotbear1110) [reply commands/duck.js]: Fixed in 4ecaa183ede7d7ffc730ff4b5c1b08666f0d613d

**Pull request merged**